src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/Component_WithEscapedParameterName/TestComponent.mappings.txt

```diff
@@ -0,0 +1,20 @@
+﻿Source Location: (20:0,20 [1] x:\dir\subdir\Test\TestComponent.cshtml)
+|1|
+Generated Location: (1005:25,20 [1] )
+|1|
+
+Source Location: (30:0,30 [1] x:\dir\subdir\Test\TestComponent.cshtml)
+|2|
+Generated Location: (1298:34,30 [1] )
+|2|
+
+Source Location: (13:0,13 [5] x:\dir\subdir\Test\TestComponent.cshtml)
+|class|
+Generated Location: (1669:46,13 [5] )
+|@clas|
```

Comment:
This looks wrong, and will likely break tooling scenarios. I thought there was a debug assert to check for this?

Need to adjust the mapping by 1 character to skip over the character that has been added here I think.

Reply:
Fixed, thanks.